site/content/docs/5.3/examples/sign-in/index.html

```diff
@@ -3,7 +3,7 @@
 title: Signin Template
 extra_css:
   - "sign-in.css"
-body_class: "text-center"
+body_class: "d-flex align-items-center py-4 bg-body-tertiary"
```

Comment:
Everything is left-aligned. If not voluntarily modified, `.text-center` should be kept here.

Reply:
This was intentional, we'll see if folks like it :).